packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -215,7 +214,6 @@ const WORKFLOW_SAVE_INTERVAL = 8000;
 const registry = new workflowService.WorkflowRegistry();
 registry.registerOp(SimulateCiemssOp);
 registry.registerOp(StratifyMiraOp);
-registry.registerOp(ModelFromCodeOp);
```

Comment:
We going to clean up the API side as well? or different PR?

Reply:
Not yet, once we redo the whole knowledge extraction